services/QuillLMS/app/models/clever_integration/teacher_classrooms_cache.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module CleverIntegration
+  class TeacherClassroomsCache
```

Comment:
Our codebase uses raw `$redis` throughout, so that was my initial choice, but now that you mentioned it, I agree.  After updating to `Rails.cache` it looks much [cleaner](https://github.com/empirical-org/Empirical-Core/pull/8653/commits/b25f5ec0bbceeee7beb0377affda073237d12057)